src/Scripting/CSharp/CSharpObjectFormatter.cs

```diff
@@ -298,7 +195,7 @@ private string FormatArrayTypeName(Type arrayType, Array arrayOpt, ObjectFormatt
                     for (int i = 0; i < rank; i++)
                     {
                         int lowerBound = arrayOpt.GetLowerBound(i);
-                        long length = arrayOpt.GetLongLength(i);
+                        int length = arrayOpt.GetLength(i);
```

Comment:
While I don't often deal with > 2GB arrays, isn't it correct to use LongLength here (or is that not portable)?